plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/modules/EmptyModuleForFolder.java

```diff
@@ -0,0 +1,73 @@
+package org.python.pydev.ast.codecompletion.revisited.modules;
+
+import java.io.File;
+
+import org.python.pydev.shared_core.string.StringUtils;
+
+public class EmptyModuleForFolder extends EmptyModule {
+
+    private static final long serialVersionUID = 1L;
+
+    public String path;
```

Comment:
I don't think you need the `path` in this case (just the name and file should suffice... I guess you just got it from the zip version where the path is needed, but that's not true in this case).